config/test.yml.erb

```diff
@@ -31,9 +31,9 @@ daemon: true
 use_pusher: <%=!ci_test%>
 
 # test environment should use precompiled, minified, digested assets like production,
-# unless it's being used for unit tests. This logic should be kept in sync with
-# the logic for setting config.assets.* under dashboard/config/.
+# unless it's being used for CI or unit tests.
 optimize_webpack_assets: <%=!ci_test%>
+optimize_rails_assets: <%=!ci_test%>
```

Comment:
Also in this change, we set this to true in locals.yml for drone runs, right? So during a drone run this will evaluate to false here, but then will get overridden to true anyway? If so it'd be clearer to use `!unit_test` instead for the condition (and update the comment also).

Reply:
true for ui tests, but not unit tests. still, removing the double negative simplifies the logic a bit.